community/contributing/docs_image_guidelines.rst

```diff
@@ -0,0 +1,132 @@
+.. _docs_image_guidelines:
+
+Creating Documentation Images
+=============================
+
+Throughout documentation images are often needed to make the explanation
+of a feature or concept as clear as possible for a reader. This page will
+explain the process from beginning to end
+
+Capturing an image
+------------------
+
+To take a picture of something in Godot a screen capture tool can be used.
+On Windows 10 and 11 that would be the Snip & Sketch program. 
+
+On macOS pressing :kbd: `Shift + Command + 3`` lets you take a screenshot
+of a portion of the screen. After pressing those keys click and drag over
+the area you wish to take a picture of. To take a picture of the entire
+screen press :kbd: `Shift + Command + 4`. All screenshots taken will
+be saved to the desktop.
+
+Each Linux desktop environment has it's own screenshot tool. For example,
+on KDE Plasma the program Spectacle is used for taking screenshots. If your
+distribution doesn't come with one by default try searching its package
+repository, or Flathub if that's supported.
+
+All screenshots should ideally be taken on a 1080p screen. Anything higher
+resolution is adding detail that doesn't make the documentation better and
+dramatically increases file size. If you're taking screenshots on a higher
+resolution screen the screenshot should be scaled down. There are instructions
+on  how to do this later in this page.
+
+Format conversion
+-----------------
+
+The current format for images in Godot's documentation is .Webp. While some
+linux programs will support saving screenshots in this format, macOS and the
+Snip & Sketch program on windows do not. For images that don't need editing
+done to them, such as precise cropping or adding outlines, Squoosh can be used.
+`Squoosh <https://squoosh.app/>`_ is a converter developed by google, is open
+source, and doesn't give google any image rights by using it. There is also a
+experimental CLI version which can be found `here <https://github.com/GoogleChromeLabs/squoosh/tree/dev/cli>`_`
```

Comment:
I think the link format needs to be fixed, and a '.' be added at the end of the sentence.